chains/manager.go

```diff
@@ -166,10 +166,10 @@ type ChainConfig struct {
 }
 
 type ManagerConfig struct {
-	StakingEnabled bool            // True iff the network has staking enabled
-	StakingCert    tls.Certificate // needed to sign snowman++ blocks
-	StakingBLSKey  *bls.SecretKey
-	TracingEnabled bool
+	SybilProtectionEnabled bool            // True iff the network has staking enabled
```

Comment:
nit: comment seems outdated and frankly not really informative. Consider dropping it

Reply:
done